ext/curl/curl_file.c

```diff
@@ -29,35 +29,41 @@
 
 PHP_CURL_API zend_class_entry *curl_CURLFile_class;
 
-static void curlfile_ctor(INTERNAL_FUNCTION_PARAMETERS)
+static void curlfile_ctor(char *fname, size_t fname_len, char *mime, size_t mime_len, char *postname, size_t postname_len, char *buffer, size_t buffer_len, INTERNAL_FUNCTION_PARAMETERS)
 {
```

Comment:
Why you need INTERNAL_FUNCTION_PARAMETERS? You don't seem to use them except for return_value. So maybe just pass return_value?